.github/workflows/bump-version.yml

```diff
@@ -49,6 +49,7 @@ jobs:
 
         re="$(sed 's/[_-]/[_-]/g' <<< "$PACKAGE")"
         sed -i "s/^$re==.*/$PACKAGE==$VERSION/g" -- requirements*.txt
+        sed -i "s/^$re>=.*/$PACKAGE>=$VERSION/g" -- requirements*.txt
```

Comment:
the original code was intentional -- the lower bound constraints should be a conscious decision rather than a compatibility update

I'm fine with this change but it wasn't what I intended with this workflow

Reply:
What is a compatibility upgrade in this case? We are intentionally bumping the schema (this is still a manual action, we don't actually have plans to auto-bump right now)